ur_robot_driver/package.xml

```diff
@@ -38,6 +38,7 @@
   <depend>ur_dashboard_msgs</depend>
   <depend>ur_msgs</depend>
   <depend>std_srvs</depend>
+  <depend>industrial_robot_status_interface</depend>
```

Comment:
Should this be a `build_depend` only?

It's a header-only library, there is nothing at runtime there.

Reply:
Yes, you're right